utils/rpms/daos.spec

```diff
@@ -0,0 +1,227 @@
+# Needed because of the GO binaries
+%undefine _missing_build_ids_terminate_build
+
+%define daoshome %{_exec_prefix}/lib/%{name}
+
+Name:          daos
+Version:       0.4.0
+Release:       2%{?dist}
+Summary:       DAOS Storage Engine
+
+License:       Apache
+URL:           https//github.com/daos-stack/daos
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: cart-devel
+BuildRequires: argobots-devel >= 1.0rc1
+BuildRequires: libpmem-devel, libpmemobj-devel
+BuildRequires: fuse-devel >= 3.4.2
+BuildRequires: protobuf-c-devel
+BuildRequires: spdk-devel, spdk-tools
+BuildRequires: fio < 3.4
```

Comment:
have we verified that spdk fio_plugin builds with this requirement?

Reply:
This is actually targeting our daos-stack/fio build.

But frankly, we should be working towards using a distro installed fio.  The complication there is in the need of the fio source to build other components and the lack of exposed API in the current fio packaging.  We should work with the fio and spdk teams to resolve that.